contracts/src/NativeSeiTokensERC20.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {IBank} from "./precompiles/IBank.sol";
+import {console2} from "forge-std/Test.sol";
+
+contract NativeSeiTokensERC20 is ERC20 {
```

Comment:
`is ERC20` gives us the sending and approval functionalities i suppose?

Reply:
yes it is the syntax to do inheritance.